routes/api/users.js

```diff
@@ -13,14 +15,12 @@ const User = require('../../models/User');
 router.post(
   '/',
   [
-    check('name', 'Name is required!')
-      .not()
-      .isEmpty(),
+    check('name', 'Name is required!').not().isEmpty(),
     check('email', 'Please include a valid email').isEmail(),
     check(
```

Comment:
Password can be more robust if we would like more security.

8 characters is a start. Some common additions are:
- at least 1 capital letter
- at least 1 special character (!?@#$%^&*()-_~)
- at least 1 number (0123456789)

These can be checked with regular expressions. Some common password regex can be found online, I think are okay to use as long as they are robust with the conditions. I don't know if there's an Express function that will do a check like that for you.